pages/guides/how-to-schedule-price-updates-with-gelato.mdx

```diff
@@ -0,0 +1,125 @@
+# How to Schedule Price Updates with Gelato
+
+This guide explains how to create an automated task that updates Pyth price feeds on a schedule
```

Comment:
"an automated Gelato task"

and I think like "on a schedule. The task will update the on-chain price after either a fixed time interval or if the price has changed by a certain percentage."